test/test_helper.exs

```diff
@@ -1 +1,26 @@
-ExUnit.start()
+ExUnit.start(capture_log: true)
+
+defmodule HelperMacros do
```

Comment:
Are you sure that you need macros?

Reply:
Maybe you can handle the helper with function instead?

Reply:
It makes more sense to me for it to be a macro - the syntax looks cleaner.

Reply:
I shall not argue about the tastes.